src/Image.php

```diff
@@ -413,131 +426,136 @@ public static function call($name, $instance, $arguments)
      * "more", but if the arg is not an image (ie. it's a constant), call
      * "more_const" instead.
      *
-     * @param mixed   $other   The right-hand argument. 
-     * @param string  $base    The base part of the operation name. 
-     * @param string  $op      The action to invoke.
-     * @param mixed[] $options An array of options to pass to the operation.
+     * @param mixed  $other   The right-hand argument.
+     * @param string $base    The base part of the operation name.
+     * @param string $op      The action to invoke.
+     * @param array  $options An array of options to pass to the operation.
      *
-     * @return mixed[] The operation result.
+     * @return Image The operation result.
```

Comment:
It'll always be Image at the moment, but it could potentially return anything. It should probaly be mixed to match `Vips::call`.